### PR TITLE
use secure urls

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/surfraw.rb [hunk 1]
```diff
@@ -20,6 +13,13 @@ class Surfraw < Formula
     sha256 "30b885dc5908318868da2739f36834ce071bc7bff1a761fdc395afe82f75efa8" => :mountain_lion
   end
 
+  head do
+    url "https://anonscm.debian.org/git/surfraw/surfraw.git"
```
Comment (DomT4) [inline Formula/surfraw.rb]: Did you test whether this still needs to not be shallow?

Comment (vszakats) [inline Formula/surfraw.rb]: Good question: No, I didn't. I'd rather put it back, if that may be an important parameter.

Comment (DomT4) [inline Formula/surfraw.rb]: Aye, some things refuse to shallow clone, or reference the git history and consequently need the fuller version. Usually one of those.

**Comment by DomT4** [comment]
Merged. Thanks again @vszakats! Appreciated as always 🐗

**Comment by DomT4** [comment]
Erm, I meant the :bow: emoji rather than 🐗, but that'll do heh.

**Pull request closed without merging**